bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -36,7 +36,7 @@ class AudioManager {
 
     const query = VoiceUsers.find({ intId: Auth.userID });
 
-    query.observeChanges({
+    query.observeChanges({ // keep track of mute/unmute in case of Flash changing it
```

Comment:
If the button is not updating accordingly the error is probably in this observer or its a server thing.
Did you check if the query state is actually updated?